tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_basic_helper.py

```diff
@@ -292,7 +293,7 @@ def __gen_traffic(testbed_config,
 
     tx_port_name = testbed_config.ports[tx_port_id].name
     rx_port_name = testbed_config.ports[rx_port_id].name
-    data_flow_rate_percent = int(100 / len(prio_list))
+    data_flow_rate_percent = int(99.98 / len(prio_list))
```

Comment:
@amitpawar12 @vmittal-msft :  Cisco recommendation is not to use 100% based on their platforms.  Is it is ok for VOQ platform or do we see any concern ? If any concern we can make it platform specific

Reply:
IMO, the change should be platform specific.

Reply:
@rraghav-cisco pls make it platform specific.